atomic_reactor/plugins/prepub_squash.py

```diff
@@ -57,14 +55,13 @@ class PrePublishSquashPlugin(PrePublishPlugin):
     is_allowed_to_fail = False
 
     def __init__(self, tasker, workflow, tag=None, from_base=True, from_layer=None,
-                 remove_former_image=True, dont_load=False):
+                 dont_load=False):
```

Comment:
Yes, I think so.

This is not actually set by default in osbs-client, but we should leave a grace period and mention it in the release notes when we release atomic-reactor next.